agent/src/vscode-shim.ts

```diff
@@ -945,12 +942,13 @@ const _commands: Partial<typeof vscode.commands> = {
 _commands?.registerCommand?.('workbench.action.reloadWindow', () => {
     // Do nothing
 })
-_commands?.registerCommand?.('setContext', (key, value) => {
+_commands?.registerCommand?.('setContext', (...args) => {
+    const [key, value] = args
     if (typeof key !== 'string') {
         throw new TypeError(`setContext: first argument must be string. Got: ${key}`)
     }
     context.set(key, value)
-    agent?.notify('window/didChangeContext', { key, value })
+    agent?.notify('window/didChangeContext', { key, value: `${value}` })
```

Comment:
As we noticed from that bug earlier, some types like Symbol can't be interpolated into a string.
```suggestion
    agent?.notify('window/didChangeContext', { key, value: value.toString() })
```